diffpy/snmf/stretchednmfapp.py

```diff
@@ -0,0 +1,26 @@
+import numpy as np
+import argparse
+from pathlib import Path
+
+
+def create_parser():
+    parser = argparse.ArgumentParser(
+        prog="stretched_nmf",
+        description="Stretched Nonnegative Matrix Factorization"
+    )
+    parser.add_argument('-i', '--input-directory', type=str, default=None,
+                        help="Directory containing experimental data. Defaults to current working directory.")
+    parser.add_argument('-o', '--output-directory', type=str,
+                        help="The directory where the results will be written. Defaults to '<input_directory/snmf_results>'.")
+    parser.add_argument('t', '--data-type', type=str, choices=['powder_diffraction', 'pdf'],
+                        help="The type of the experimental data.")
+    parser.add_argument('-l', '--lift', type=float, default=1,
```

Comment:
I would call it `lift-factor` for me the lift is how much it is lifted.  the lift-factor is how much I multiply the min by

Reply:
Ok. I will rename it